src/components/IOUConfirmationList.js

```diff
@@ -111,12 +115,12 @@ class IOUConfirmationList extends Component {
 
                 // Convert from cent to bigger form
                 // USD is temporary and there must be support for other currencies in the future
-                `$${this.calculateAmount(true) / 100}`,
+                `${this.props.selectedCurrency.currencySymbol}${this.calculateAmount(true) / 100}`,
```

Comment:
We recently merged support for localization/internationalization in the app, that means all texts, dates and amounts need to use the locale aware methods. So everywhere in this PR we should do that. In order to do it you use the `withLocalize` HOC and then call numberFormat passing `style: 'currency, currency: this.props.selectedCurrency.currencySymbol` in the options

Reply:
Looks like this still needs to be done?

Reply:
Oops sorry. This was missed. I will resolve it.